core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -1030,10 +1030,18 @@ func (ec *EthConfirmer) handleInProgressAttempt(ctx context.Context, etx EthTx,
 			return errors.Wrap(err, "could not bump gas for terminally underpriced transaction")
 		}
 		promNumGasBumps.WithLabelValues(ec.chainID.String()).Inc()
-		ec.lggr.Errorw(
-			fmt.Sprintf("gas price %v wei was rejected by the eth node for being too low. Eth node returned: '%s'",
-				attempt.GasPrice.String(), sendError.Error()),
-			"previousAttempt", attempt, "replacementAttempt", replacementAttempt)
+		ec.lggr.With(
+			"sendError", sendError,
+			"maxGasPriceConfig", ec.config.EvmMaxGasPriceWei(),
+			"previousAttemptGasPrice", attempt.GasPrice,
+			"previousAttemptGasFeeCap", attempt.GasFeeCap,
+			"previousAttemptGasTipCap", attempt.GasTipCap,
+			"replacementAttemptGasPrice", replacementAttempt.GasPrice,
+			"replacementAttemptGasFeeCap", replacementAttempt.GasFeeCap,
+			"replacementAttemptGasTipCap", replacementAttempt.GasTipCap,
+		).
+			Errorf("gas price %s wei was rejected by the eth node for being too low. Eth node returned: '%s'",
+				attempt.GasPrice.String(), sendError.Error())
```

Comment:
Codecov is reporting this branch is untested, do you think you might be able to add a case for this?

Reply:
Yup, will do.